skree/README.md

```diff
@@ -0,0 +1,64 @@
+###### Credit for this guide goes to steamport, from [Sunset Station] (https://github.com/SunsetStation/SunsetStation/blob/master/sunset/README.md). Edits done by battletrap360.
+
+# Introduction to the "skree" Folder
+
+## Why do we use it?
+
+To keep up-to-date with /tg/station while trying to avoid merge conflicts as much as possible, we modularize everything. This is the same method [HippieStation](https://github.com/HippieStation/HippieStation/tree/master/hippiestation) uses. The more code in the "skree" folder, the fewer conflicts will exist in future updates.
+
+## What does it mean to modularize something?
+
+Something is modular when it exists independent from the rest of the code. This means that by simply adding something modular to the DME file, it will exist in-game. It is not always possible to completely modularize something, but if standards are followed correctly, then there should be few to none conflicts with /tg/station in the future.
+
+## Please mark your changes
+
+All modifications to non-skree files should be marked.
+
+- Multi line changes start with `// skree start` and end with `// skree end`
+- You can put a messages with a change if it isn't obvious, like this: `// skree start - reason`
+  - Should generally be about the reason the change was made, what it was before, or what the change is
+  - Multi-line messages should start with `// skree start` and use `/* Multi line message here */` for the message itself
+- Single line changes should have `// skree` or `// skree - reason`
+
+If you need to mirror a file, or function into a yog-specific file, please leave behind a comment stating where it went.

```

Comment:
yog specific?

Reply:
It is probably something we won't need to worry about, I ripped the guide from steamport, a dude who has way more coding smarts than me